middleware/queue/include/queue/common/ipc/message.h

```diff
@@ -795,6 +796,59 @@ namespace casual
                };
                
             } // remove
+            namespace recovery
+            {
+               enum class Type : int
+               {
+                  commit,
+                  rollback
+               };
+
+               inline std::ostream& operator << ( std::ostream& out, recovery::Type value)
+               {
+                  switch( value)
+                  {
+                     using Enum = recovery::Type;
+                     case Enum::commit: return out << "commit";
+                     case Enum::rollback: return out << "rollback";
+                  }
+                  return out << "<unknown>";
+               }
```

Comment:
Change to: 
`inline constexpr std::string_view description( Type value) noexcept` 
(then we can get rid of ostream and the includes all together).